scripts/update_qm_list.py

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2017 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+import hera_mc.mc as mc
+
+parser = mc.get_mc_argument_parser()
+args = parser.parse_args()
+db = mc.connect_to_mc_db(args)
+session = db.sessionmaker()
+
+metric_list = copy(ant_metric_list)
```

Comment:
Woops, those lines were actually from an old version. The metric lists are now imported in the  `update_qm_list` method of mc_session. I will fix this.